src/modules/job-list/job_state.c

```diff
@@ -1146,6 +1146,22 @@ static int depthfirst_map (struct list_ctx *ctx, const char *key,
     return rc;
 }
 
+/* Call zlistx_sort() on the list, then reacquire the list handles since
```

Comment:
commit message typo "referend" -> "referenced"?

Reply:
I think we all agree that this `zlistx` API choice was a massive foot-gun!